src/network/netdev/tunnel.c

```diff
@@ -1190,6 +1196,7 @@ static void netdev_tunnel_init(NetDev *netdev) {
 
         t->local_type = _NETDEV_LOCAL_ADDRESS_TYPE_INVALID;
         t->pmtudisc = true;
+        t->ignore_df = false;
```

Comment:
Please drop unnecessary initialization. The object is already initialized with zero.

Reply:
done